components/pages/UserStories/UserStories.js

```diff
@@ -65,7 +65,24 @@ function progressWidth(storyIndex, progressBarIndex, progress) {
 
 export function UserStories({ userId }) {
   const [dimensions, setDimensions] = React.useState({ width: 0, height: 0 });
-  const storiesData = useSelect((select) => select('ig-stories').getStories());
+  const areStoriesResolved = useSelect((select) =>
+    select('ig-stories').hasFinishedResolution('getStories')
+  );
+  const [isOneUser] = React.useState(!areStoriesResolved);
+
+  const storiesData = useSelect(
+    (select) => {
+      if (areStoriesResolved) {
+        return select('ig-stories').getStories();
+      } else {
+        if (!userId) {
+          return [];
+        }
+        return select('ig-stories').getStories(userId);
+      }
+    },
+    [userId]
+  );
```

Comment:
This belongs to the stories PR. To remove change, copy the code from master and move this code to https://github.com/alshakerM/my-instagram/pull/55